lib/modules/manager/sveltos/util.ts

```diff
@@ -0,0 +1,13 @@
+import { regEx } from '../../../util/regex';
+
+export function removeRepositoryName(
+  repositoryName: string,
+  chartName: string,
+): string {
+  const repoNameWithSlash = regEx(`^${repositoryName}/`, undefined, false);
+  let modifiedChartName = chartName.replace(repoNameWithSlash, '');
```

Comment:
```suggestion
  let modifiedChartName = chartName.substring(repoName.length). replace (regEx('^/'), '');
```
why not even simpler?